core/domain/opportunity_jobs_one_off.py

```diff
@@ -1,147 +0,0 @@
-# coding: utf-8
-#
-# Copyright 2016 The Oppia Authors. All Rights Reserved.
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#      http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS-IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-
-"""One-off jobs related to opportunity models."""
-
-from __future__ import absolute_import  # pylint: disable=import-only-modules
-from __future__ import unicode_literals  # pylint: disable=import-only-modules
-
-import ast
-import copy
-
-from core import jobs
-from core.domain import opportunity_services
-from core.platform import models
-
-(opportunity_models, topic_models, skill_models,) = (
-    models.Registry.import_models([
-        models.NAMES.opportunity, models.NAMES.topic, models.NAMES.skill]))
-
-
-class ExplorationOpportunitySummaryModelRegenerationJob(
```

Comment:
This job is essential and required every time we add a new language or change the functionality to generate/evaluate the opportunity.

Reply:
+1.

Reply:
Ack'd, I'll increase its priority in our tracker. Regardless, it will need to be deleted in this PR.